code/modules/projectiles/guns/ballistic/pistol.dm

```diff
@@ -130,6 +132,23 @@
 /obj/item/gun/ballistic/automatic/pistol/v38/less_lethal
 	starting_mag_type = /obj/item/ammo_box/magazine/v38/rubber
 
+/obj/item/gun/ballistic/automatic/pistol/v38/less_lethal/Initialize(mapload)
+	if(GLOB.does_howard_exist || prob(90))
+		return ..()
+	GLOB.does_howard_exist = TRUE
+	new/obj/item/gun/ballistic/automatic/pistol/v38/less_lethal/howard(loc)
```

Comment:
deleting the item and replacing it in its loc will not always work; a less destructive method is to reskin the item, refresh its overlays, and store its state in a variable like `is_howard` and then handle deletion logic (global FALSE set) if that is true

Reply:
They only come in crates, handling it in initialize is robust enough that it will handle crates, admin spawning, and spawning in containers